Rapid7 - Threat Command (IntSights) pack release 3.1.0 (#30954)
demisto/content: Packs/IntSight/ReleaseNotes/3_1_0.md

#### Incident Fields

> - **Rapid7 ThreatCommand CSV**
>   - Added the more columns.

Reviewer: ```suggestion
- **Rapid7 ThreatCommand CSV**
  - Added more columns.
```

#### Incident Types

- **Rapid7 ThreatCommand Alert**

>   - Updated the incident type to be attached to the playbook.

Reviewer: ```suggestion
  - Updated the default playbook for the incident type.
```
